OSVR-Unity/Assets/OSVRUnity/src/VREye.cs

```diff
@@ -247,20 +249,31 @@ public void CreateSurfaces(uint surfaceCount)
 
                     //distortion
                     bool useDistortion = Viewer.DisplayController.DisplayConfig.DoesViewerEyeSurfaceWantDistortion(Viewer.ViewerIndex, (byte)_eyeIndex, surfaceIndex);
-                    if(useDistortion)
+                    useDistortion |= !displayController.UseRenderManager;
+
+                    if (useDistortion)
                     {
                         //@todo figure out which type of distortion to use
                         //right now, there is only one option, SurfaceRadialDistortion
                         //get distortion parameters
-                        OSVR.ClientKit.RadialDistortionParameters distortionParameters =
-                        Viewer.DisplayController.DisplayConfig.GetViewerEyeSurfaceRadialDistortion(
-                        Viewer.ViewerIndex, (byte)_eyeIndex, surfaceIndex);
+                        OSVR.ClientKit.RadialDistortionParameters distortionParameters;
+
+                        try
+                        {
+                            distortionParameters = Viewer.DisplayController.DisplayConfig.GetViewerEyeSurfaceRadialDistortion(Viewer.ViewerIndex, (byte)_eyeIndex, surfaceIndex);
```

Comment:
For the moment, I would not recommend using the DisplayConfig API - I don't know if it's up to date with how the RenderManager works - if I recall correctly, the code that parses the display config and render manager configs is currently forked in RenderManager and OSVR-Core. Eventually that needs to be moved to OSVR-Core and removed from RenderManager (so that RM calls into OSVR-Core), but in the meantime look out for issues with the OSVR-Core DisplayConfig.